e2e/wdio/core/tests/wizard.e2e-spec.ts

```diff
@@ -205,7 +213,7 @@ describe('Wizard component test', () => {
     function checkReOpen(section: string, block: string): void {
         click(section + button);
         if (section === defaultExample) {
-            waitForElDisplayed(block + nextStep);
+            pause(5000)
```

Comment:
maybe we should keep the waitForElDisplayed here after the pause, just to make sure everything is ready for the click.